packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -22,20 +22,30 @@ export interface TreeHeaderProps extends HasLabel {
 	inlineValue?: React.ReactElement | string;
 
 	// TODO: metadata
+	metadata?: string | undefined;
 }
 
 /**
  * Renders the header of the item.
  */
 export function TreeHeader(props: TreeHeaderProps): React.ReactElement {
-	const { label, nodeTypeMetadata, inlineValue } = props;
+	const { label, nodeTypeMetadata, inlineValue, metadata } = props;
 
 	return (
 		<div style={{ width: "auto" }}>
 			{`${label}`}
 			<span style={{ color: tokens.colorPaletteRedBorderActive, fontSize: "10px" }}>
 				{nodeTypeMetadata === undefined ? "" : ` (${nodeTypeMetadata})`}
 			</span>
+			<span
+				style={{
+					color: tokens.colorPalettePlatinumBorderActive,
```

Comment:
Just want to make doubly sure - have you tested this in our high-contrast mode? Does it look okay there?